src/blog/2023/08/flowfuse-1-11-release.md

```diff
@@ -0,0 +1,80 @@
+---
+title: FlowFuse 1.11 makes it easier to get started with FlowFuse and Node-RED
+subtitle: The new FlowFuse 1.11 release includes a new starter tier for FlowFuse Cloud, Personal Access Tokens for API access and improvements to device management.
+description: The new FlowFuse 1.11 release includes a new starter tier for FlowFuse Cloud, Personal Access Tokens for API access and improvements to device management.
+date: 2023-08-31 
+authors: ["ian-skerrett"]
+image: /blog/2023/08/images/release-1-11-graphic.png
+tags:
+    - posts
+    - flowfuse
+    - releases
+---
+
+FlowFuse 1.11 introduces a new starter tier for FlowFuse Cloud that makes it easier to get started with FlowFuse and Node-RED.
+
+<!--more-->
+## New FlowFuse Cloud Starter Tier  [#2328](https://github.com/flowforge/flowforge/issues/2328)
+
+It is now easier for Node-RED developers to get started with FlowFuse and Node-RED. The new starter tier allows developers to use two Node-RED instances and two remote device deployments. Ideal for creating proof of concepts or running a home automation system with Node-RED. 
+
+FlowFuse provides a cloud hosted version of Node-RED so developers don't need to worry about Node-RED installation or operation. This makes it a lot easier to get started with Node-RED and easier to maintain a running instance. 
+
+
+## FlowFuse API access now possible via personal access tokens [#14](https://github.com/flowforge/flowforge/issues/14)
+FlowFuse APIs are now accessible via personal access tokens (PAT). This makes it possible to create automation scripts that interact with the FlowFuse platform using the API and authenticate the scripts with the PAT. 
+
+## Usability Improvements to Device Management [#2294](https://github.com/flowforge/flowforge/issues/2334)
+
+A number of usability improvements have been added to the FlowFuse device management solution to make it more flexible and intuitive to use. These improvements include being able to associate devices at the application level allowing for easier editing of Node-RED instances on edge devices.
+
+## FlowFuse Rebranding [#119](https://github.com/orgs/flowforge/projects/1?pane=issue&itemId=34719640)
+
+Earlier in August, [FlowForge announced](https://flowfuse.com/blog/2023/08/flowforge-is-now-flowfuse/) a change to our company and product name to FlowFuse. Work has begun to change the product branding to FlowFuse. The UI has been rebranded and the remaining points will be changed in the next release.
```

Comment:
The URL changes too today, I think it should be mentioned

Reply:
https://github.com/flowforge/admin/issues/208

It's a separate configuration step after the release. Old instances will retain their previous domain configurations. Newly created instances will be created under the new domain.

https://flowforgeworkspace.slack.com/archives/C032Q63FGG1/p1693477042121479

Both https://app.flowforge.com/ and https://app.flowfuse.com/ will work.

Reply:
Can you help draft some words on this topic? It is not clear to me what an end use needs to do. Is this in our new documentation that we can link to?

Reply:
There will be no domain change today, it will remain unchanged https://app.flowforge.com/

Reply:
Closing conversation as resolved. We won't add URL change to the blog post.